interp/constrintern.ml

```diff
@@ -1369,7 +1369,7 @@ let sort_fields ~complete loc fields completer =
                (* the order does not matter as we sort them next,
                   List.rev_* is just for efficiency *)
                let remaining_fields =
-                 let complete_field (idx, _field_ref) = (idx, completer idx) in
+                 let complete_field (idx, _field_ref) = (idx, completer idx _field_ref base_constructor) in
```

Comment:
In the definition of `base_constructor` you have `record.Recordops.s_CONST` of type `Names.constructor`, I think this is what should be passed.

Reply:
How do I print a `Names.constructor`? Or, a better question, how do I _discover_ how to print `Names.constructor`?